README.md

```diff
@@ -65,6 +65,9 @@ Enables you to create more customized BLE Peripheral's compared to the basic UAR
    * LC
  * [MPIDE](http://chipkit.net/started/install-chipkit-software/) ([ChipKit](http://chipkit.net))
    * PIC32MX
+ * [Arduino](https://www.arduino.cc/en/Main/arduinoBoardMega) ([Arduino Mega](https://www.arduino.cc/en/Main/arduinoBoardMega))
+   * Arduino MEGA
+   * You may experience memory problems on Arduino boards with 32kb flash storage (e.g. Uno,Leonardo)
```

Comment:
Well, there are 2 concerns, flash space and RAM/SRAM.

The Uno has 32kB of flash space, and only 2kB of SRAM, Leonardo has 2.5kB of SRAM, so a bit more room for user variables. I think your concern is more about RAM correct?